docs/cncf/cncf_testing.md

```diff
@@ -0,0 +1,88 @@
+this document are the recommended way to configure a non-production two-node
+setup to be used **only** for running the CNCF conformance tests.
+
+## Prerequisites
+A physical `hypervisor host` with the [libvirt](https://libvirt.org/) virtualization
+platform to be used for bootstrapping MicroShift primary and secondary hosts for
+running the CNCF conformance tests.
+
+Depending on the desired MicroShift installation method, follow the instructions
+to set up one of the configurations for the primary and secondary hosts:
+* [RHEL with MicroShift RPMs](./cncf_config_rpm.md)
+* [MicroShift on RHEL for Edge](./cncf_config_r4e.md)
+
+Log into the `hypervisor host` and set the environment variables that denote
+primary and secondary host names and IP addresses to be used in the subsequent
+commands.
+```
+PRI_HOST=microshift-cncf-pri
+PRI_ADDR=192.168.122.118
+
+SEC_HOST=microshift-cncf-sec
+SEC_ADDR=192.168.122.70
+```
+
+## Configure Primary and Secondary Hosts
+Run the following commands to copy the configuration script to the primary host
+and run it remotely.
+```
+git clone https://github.com/openshift/microshift.git ~/microshift
+cd ~/microshift/
+scp -o StrictHostKeyChecking=no ./scripts/cncf/configure-pri.sh redhat@${PRI_ADDR}:
+ssh redhat@${PRI_ADDR} ./configure-pri.sh "${PRI_HOST}" "${PRI_ADDR}" "${SEC_HOST}" "${SEC_ADDR}"
+```
+
+If the configuration script runs successfully, it prints the list of the
+`kubelet` configuration files that need to be copied to the secondary host.
+
+Copy the `kubelet` configuration files from the primary to the secondary host.
+```
+scp -o StrictHostKeyChecking=no \
+    redhat@${PRI_ADDR}:/home/redhat/kubelet-${SEC_HOST}.{key,crt} \
+    redhat@${PRI_ADDR}:/home/redhat/kubeconfig-${PRI_HOST} \
+    redhat@${SEC_ADDR}:
+```
+
+Run the following commands to copy the configuration script to the secondary host
+and run it remotely.
+```
+cd ~/microshift/
+scp ./scripts/cncf/configure-sec.sh redhat@${SEC_ADDR}:
+ssh redhat@${SEC_ADDR} ./configure-sec.sh "${PRI_HOST}" "${PRI_ADDR}" "${SEC_HOST}" "${SEC_ADDR}"
+```
+
+## Run CNCF Conformance Tests
+Run the following commands to download and execute the CNCF conformance tests
+using the primary and secondary MicroShift instances.
+
+> See the [How to submit conformance results](https://github.com/cncf/k8s-conformance/blob/master/instructions.md)
+> document for more information on the CNCF Kubernetes conformance tests.
+
+```
+export KUBECONFIG=$(mktemp /tmp/microshift-kubeconfig.XXXXXXXXXX)
+scp -o StrictHostKeyChecking=no \
+    redhat@${PRI_ADDR}:/home/redhat/kubeconfig-${PRI_HOST} \
+    ${KUBECONFIG}
+
+go install github.com/vmware-tanzu/sonobuoy@latest
+~/go/bin/sonobuoy run --wait \
+    --mode=certified-conformance \
+    --dns-namespace=openshift-dns \
+    --dns-pod-labels=dns.operator.openshift.io/daemonset-dns=default
+~/go/bin/sonobuoy retrieve
+
+rm -f ${KUBECONFIG}
+```
+
+> Before running the tests, make sure that the `microshift-cncf-pri` host name
```

Comment:
Also, make sure all the pods are running in both the nodes before running the tests.

Reply:
Added more prerequisites before running the tests.